src/Table/__stories__/Table.stories.js

```diff
@@ -47,21 +59,9 @@ storiesOf('Table', module)
       },
     ];
 
-    const compressedRows = boolean('compressedRows', false, 'State');
-    const noZebra = boolean('No Zebra', false, 'State');
-    const options = {
-      top: 'top',
-      bottom: 'bottom',
-    };
-    const headerPosition = select('Header position', options, 'top', 'State');
-
     return (
-      <Table
-        data={data}
-        colsDef={getColsDef()}
-        compressedRows={compressedRows}
-        header={headerPosition}
-        noZebra={noZebra}
-      />
+      <div style={{ background: 'white', textAlign: 'left' }}>
```

Comment:
Why `textAlign: 'left'`? It does not seem the component style changes without it

Reply:
Just s simple copy/past 😇